applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.h

```diff
@@ -67,9 +67,9 @@ struct le_audio_encoded_audio {
 };
 
 struct stream_index {
-	uint8_t level1_idx; /* BIG / CIG */
-	uint8_t level2_idx; /* Subgroups if applicable */
-	uint8_t level3_idx; /* BIS / CIS */
+	uint8_t lvl1_idx; /* BIG / CIG */
+	uint8_t lvl2_idx; /* Subgroups if applicable */
+	uint8_t lvl3_idx; /* BIS / CIS */
```

Comment:
It was already taking a lot of code space, having them written out like this makes things more difficult to read because they are forced to be split over multiple lines. I find that harder to read than to into the header file to see what the different levels are.